src/jvm.jl

```diff
@@ -152,6 +152,10 @@ end
 addOpts(s::String) = isloaded() ? @warn("JVM already initialised. This call has no effect") : push!(opts, s)
 
 function init()
+    if get(ENV, "JULIA_COPY_STACKS", "") ∉ ("1", "yes")
```

Comment:
If we are doing this, I'd like this to be a little more sophisticated wrt Julia versions. `JULIA_COPY_STACKS` is not needed on `v1.0` and does not work on `v1.1` or `v1.2`

Reply:
I've added a version number check.